lala_bar/src/music_bar.rs

```diff
@@ -59,9 +63,15 @@ impl LalaMusicBar {
         let week = date.format("%A").to_string();
         let time = self.datetime.time();
         let time_info = time.format("%H:%M").to_string();
-
+        let mut week_btn = button(text(week.clone())).on_press(Message::NewWindowRight);
```

Comment:
make on_press to CalanderToggle,

if there is already a calander, close, else , open it